test/functional/cmdLineTests/runtimemxbeanTests/src/GetPid.java

```diff
@@ -24,11 +24,14 @@
 import java.lang.management.ManagementFactory;
 
 class GetPid {
-
-	public static void main(String[] args) {
-		RuntimeMXBean mb = ManagementFactory.getRuntimeMXBean();
-		long id = mb.getPid();
-		System.out.println(id);
-	}
-
+        public static void main(String args[]) {
+                        try {
+                                RuntimeMXBean mb = ManagementFactory.getRuntimeMXBean();
+                                long id = mb.getPid();
+                                System.out.println(id);
+                                /* Puts Java process on hold until system receives output from testing script getPidTest.pl*/
+                                System.in.read();
+                        } catch (Exception e) {
```

Comment:
some issues with indenting here, you can use the eclipse auto-format to fix this